### PR TITLE
Fix #953: Make it clearer how to contact the moderation team.

### LINKED ISSUE #953: Code of Conduct's instructions on contacting a moderator don't work well
(mostly copied from [Gitter](https://gitter.im/scala/center?at=5b9c36b8d655361f76feae3a))

When discussing how to contact moderators, the [Code of Conduct](https://scala-lang.org/conduct/) links to the 'about' page on Contributors, and to the Scala Center website. The latter does not have any way to privately message member of the Scala Center at all (barring the twitters of some of the developers, which I would be reticent to DM). The 'about' page on Contributors seems initially useful, but really isn't great:

- most people are listed under "Admins" rather than "Moderators", and it is perhaps not 100% clear that the former is a subset of the latter (Admins could potentially just be people who take care of logistical stuff to make sure the site stays running, but don't moderate)
- You can't PM anyone until you get the 'Basic' badge, which means that you have to do a whole bunch of interaction on Contributors (reading threads, and perhaps other stuff? the exact requirements aren't clear to me) before you can contact a moderator about your issue. At least personally, I didn't get the badge until a month after joining (you shouldn't even need to sign up for the forum at all in order to contact a moderator, but honestly even lifting the 'Basic' restriction is much better than what we have now)
- it might also be preferable to have a way to contact moderators without having to choose a specific person from a whole list of people. I don't think we should remove the option to choose a specific person to contact, but it would be nice to also be able to just "contact the moderation team"

### PR DESCRIPTION
**Pull request by sjrd**
Also /cc @darjutak 

@adriaanm @SethTisue @darjutak Tell me if it's fine for you to be listed there.

### CODE REVIEW

**Comment by SethTisue** [comment]
@NthPortal lgty?

**Comment by SethTisue** [comment]
was just taking another look, maybe

> To report any violation of this Code of Conduct on the channels listed above

should be

> For CoC-related questions or to report violations,

so it's clear it isn't _only_ to report violations.

**Comment by sjrd** [comment]
done

File: conduct.md [hunk 1]
```diff
@@ -77,6 +76,22 @@ those projects for enforcement. If you wish to use this code of conduct for your
...
+### Contact
+
+For CoC-related questions or to report possible violations on the channels
+listed above,
+
+* contact one of the moderators active on that channel if you can identify
+  them, or
+* send an e-mail to [Scala-CoC@epfl.ch](mailto:Scala-CoC@epfl.ch), which
+  forwards to general moderators at the Scala Center, or
+* send an e-mail to one of the following general moderators if for some reason
```
Comment (NthPortal) [inline conduct.md]: I might get rid of the 'for some reason'. thoughts?

```suggestion
* send an e-mail to one of the following general moderators if
```

Comment (SethTisue) [reply conduct.md]: agree, "for some reason" has a faint (though, I'm sure, unintended) air of eye-rolling 🙄

Comment (sjrd) [reply conduct.md]: Indeed. Done.

**Comment by SethTisue** [comment]
I've also just submitted https://github.com/scala/scala-lang/pull/1014, a small CoC change that is related to this.

**Comment by SethTisue** [comment]
merging since it's been sitting a while. but I've also emailed Adriaan & Darja to double check

**Pull request merged**